.changeset/strong-spiders-float.md

```diff
@@ -0,0 +1,6 @@
+---
+'@launchpad-ui/icons': minor
+'@launchpad-ui/core': minor
```

Comment:
```suggestion
'@launchpad-ui/icons': patch
'@launchpad-ui/core': patch
```
Thanks for the contribution! We use[ 0.x versioning](https://semver.org/spec/v0.1.0.html) so these will be patches.